docs/epytext_demo/constants.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module demonstrating the constant representations.
+"""
+
+import re
+from .demo_epytext_module import demo_fields_docstring_arguments, _PrivateClass
+
+A_DICT = {'1':33, '2':[1,2,3,{7:'oo'*20}], 
+         '3': demo_fields_docstring_arguments, 
+         '4': _PrivateClass.method_inside_private, 
+         '5': re.compile('^<(?P<descr>.*) at (?P<addr>0x[0-9a-f]+)>$') }
+"""
+Pydoctor does a pretty good job at analyzing constant values ;-)
+"""
+
+A_LIST = [1,2,[5,6,[(11,22,33),9],10],11]+[99,98,97,96,95]
+
+ANOTHER_LIST = list(range(100))
+
+RE_STR = re.compile("(foo (?P<a>bar) | (?P<boop>baz))")
+
+RE_RAW_STR = re.compile(r"abc_raw \t\r\n\f\v \xff \uffff")
+
+RE_WITH_UNICODE = re.compile("abc 😀")
```

Comment:
Maybe add a docstring that unicode is supported.

it would be nice to have a docstring for each of these exemplified constants to describe the presented functionality.

For example, I am not sure what is the different between `RE_RAW_STR` and `RE_RAW_STR2`

Also,  I see `A_LIST` which I guess it exemplified that you can have nested lists.
And `ANOTHER_LIST` which exemplified that you can have generated or constructed lists.